### ISSUE TITLE
Docker image crashes on load with GLIBC warning

### ISSUE THREAD

**Issue by jjd314** [RedisLabsModules/redismod#25]
The docker image pushed today (digest 4c65a20afb15) fails to start for me with this error:

`Module /usr/lib/redis/modules/redisearch.so failed to load: /lib/x86_64-linux-gnu/libm.so.6: version 'GLIBC_2.29' not found (required by /usr/lib/redis/modules/redisearch.so)`

I foolishly deployed with ':latest' still in the spec, and now my production site is wedged.  Any workaround advice is appreciated. If anyone knows the sha of the image just before this one, please post it.

**Comment by jjd314** [comment]
I tried to restart with the most recent image I could find in my local docker repo, sha 2be634e2fd2476df4583eb0914aa49a9f5859f7a7c07c7a55403fd81681526b6, but it would not load my RDB file.  In desperation I copied the version of /usr/lib/redis/modules/redisearch.so out of that image and built a new image of my own based on redislags/redismod:latest and just plopped that .so file on top.  I suspect that I will find out that doing so is what is known in the business as a Very Bad Idea, but at the moment it seems to have got my installation back online.

**Comment by szho42** [comment]
we also had the same issue,
"Module /usr/lib/redis/modules/redisearch.so failed to load: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /usr/lib/redis/modules/redisearch.so)"

I guess it might be worth to keep more previous release on dockerhub, in case the latest docker build has issues.

**Comment by guyroyse** [comment]
We have a Discord user who is having this same problem as well.

**Comment by chayim** [comment]
Oy, I'm sorry this happened!  First off - I just validated with the redislabs/redismod:edge, and that container works. However, it's literally the tip of the edge. I just pushed up [this PR](https://github.com/RedisLabsModules/redismod/pull/26) to fix this issue properly, for the latest link.

Were you to clone that pull and run a _docker build ._ in the root of your clone, you will receive a working docker - if you're blocked.
